trex-container-app/server/Dockerfile

```diff
@@ -60,6 +60,22 @@ WORKDIR /opt/trex/trex-core/scripts
 ENV TREX_DIR="/opt/trex/trex-core/scripts"
 ENV TRAFFICGEN_DIR="/opt/trafficgen"
 
+# Create custom user to avoid using root account
+RUN useradd example-cnf
+
+# This is to be able to manage some files that belong to root account
+RUN usermod -a -G root example-cnf
+
+# Allow example-cnf to use sudo permissions without asking for password
+RUN echo -e "example-cnf\tALL=(ALL)\tNOPASSWD: ALL" > /etc/sudoers.d/example-cnf
+
+# Make sure /etc/trex_cfg.yaml is present and with proper permissions
+RUN touch /etc/trex_cfg.yaml
+RUN chmod 664 /etc/trex_cfg.yaml
```

Comment:
Ramon, Just curious what is this file for?

Reply:
It's a file that is created in this script: https://github.com/openshift-kni/example-cnf/blob/main/trex-container-app/server/scripts/trex-cfg-configure#L34, and since it's under `/etc/`, if we don't do anything, we would need root permissions to create it. My solution was to update the group permissions, so that the example-cnf user, that belongs to the root group, can create the file

Reply:
Thanks for the clarification